api/context.md

```diff
@@ -41,26 +41,35 @@ new Response('Thank you for coming', {
 })
 ```
 
-## text()
-
-Render text as `Content-Type:text/plain`.
-
+## body()
+Return the HTTP response.
 ```ts
-app.get('/say', (c) => {
-  return c.text('Hello!')
+app.get('/welcome', (c) => {
+  return c.body('Thank you for coming')
```

Comment:
Though we can specify it with `c.header,` when you use `c.body,` we should add `Content-Type.` For text, `text/plain` is automatically attached without `Content-Type`, but it is not recommended.